app/models/foreman_openscap/arf_report.rb

```diff
@@ -164,11 +164,12 @@ def equal?(other)
     end
 
     def destroy
-      if openscap_proxy_api.destroy_report(self, ForemanOpenscap::Helper::find_name_or_uuid_by_host(host))
-        super
-      else
-        false
+      begin
+        openscap_proxy_api.destroy_report(self, ForemanOpenscap::Helper::find_name_or_uuid_by_host(host))
+      rescue => e
```

Comment:
Should this rescue be more specific or are we permissive with the errors we're rescuing here?